pkg/cmd/cli/describe/deployments.go

```diff
@@ -73,6 +73,15 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, deploymentConfig.ObjectMeta)
+		var (
+			deploymentsHistory   []kapi.ReplicationController
+			activeDeploymentName string
+		)
+		if d.config == nil {
+			if rcs, err := d.kubeClient.ReplicationControllers(namespace).List(kapi.ListOptions{LabelSelector: labels.Everything()}); err == nil {
```

Comment:
Why are we listing everything? Can you change this to list only the controllers we want? (like [here](https://github.com/openshift/origin/blob/12dab60b3f5d97dd46e0cb3ba716ce0e28fc6484/pkg/deploy/controller/deploymentconfig/controller.go#L86-L87))